src/devices/tuya.ts

```diff
@@ -19008,25 +19019,45 @@ export const definitions: DefinitionWithExtend[] = [
         vendor: "Tuya",
         description: "Dimmer knob with two lights",
         fromZigbee: [tuya.fz.datapoints],
-        toZigbee: [tuya.tz.datapoints],
+        toZigbee: [tzLocal.TS0601_knob_dimmer_switch_group_id, tuya.tz.datapoints],
         exposes: [
+            e.switch(),
             e.switch().withEndpoint("l1"),
             e.switch().withEndpoint("l2"),
-            e.light_brightness_colortemp([154, 370]),
-            e.enum("adjustment_mode", ea.STATE_SET, ["brightness", "color_temp"]).withDescription("Adjustment mode"),
+            tuya.exposes.switchMode().withEndpoint("l1"),
+            tuya.exposes.switchMode().withEndpoint("l2"),
+
+            e.light_brightness_colortemp([2700, 6500]),
```

Comment:
```suggestion
            e.light_brightness_colortemp([2700, 6500]),
```

This looks like the Kelvin range, not the mireds wich it should be, the old `e.light_brightness_colortemp([154, 370]),` looks more realistic

Reply:
The device itself displays the color temperature in Kelvins, so I thought that the converter should reflect what's shown on the display. Does it make sense? Or the mired range should be used anyway, for some other reasons?
![20251007_225148](https://github.com/user-attachments/assets/f826082f-7338-4880-9da0-8a243184ceee)
![20251007_225159](https://github.com/user-attachments/assets/cb40f999-44ed-4cdc-909a-2f90790d2200)